### ISSUE TITLE
com.kumuluz.ee:kumuluzee-maven-plugin:3.9.0:repackage failed.: UnsupportedOperationException

### ISSUE THREAD

**Issue by Karm** [eclipse/microprofile-starter#443]
GitHub CI keeps failing for KumuluzEE on:

```
[INFO] Copying jetty-server-9.4.15.v20190215.jar to /home/runner/work/_temp/kumuluzee/service-a/target/classes/lib/jetty-server-9.4.15.v20190215.jar
[INFO] Copying guava-27.1-jre.jar to /home/runner/work/_temp/kumuluzee/service-a/target/classes/lib/guava-27.1-jre.jar
[INFO] /home/runner/work/_temp/kumuluzee/service-a/src/main/webapp
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 2 resources
[INFO] Unpacking kumuluzee-loader dependency.
[INFO] Building jar: /home/runner/work/_temp/kumuluzee/service-a/target/kumuluzee-uber.jar
[INFO] Repackaging jar: /home/runner/work/_temp/kumuluzee/service-a/target/kumuluzee.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.883 s
[INFO] Finished at: 2021-09-08T21:25:20Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.kumuluz.ee:kumuluzee-maven-plugin:3.9.0:repackage (package) on project kumuluzee-service-a: Execution package of goal com.kumuluz.ee:kumuluzee-maven-plugin:3.9.0:repackage failed.: UnsupportedOperationException -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

I cannot reproduce locally. Hmm...

**Comment by Emily-Jiang** [comment]
I hit the same issue. I tried to update the plugin version to 3.12.1, which did not fix the issue. Should we disable the tests for KumuluzEE for the time being as I need to get my fix in so that I can do a demo tomorrow?

**Comment by Karm** [comment]
Solved by enforcing Maven version 3.6.3

**Comment by ozkanpakdil** [comment]
Hi, @Karm I am having a similar error in https://github.com/ozkanpakdil/test-microservice-frameworks 

**Error:  Failed to execute goal com.kumuluz.ee:kumuluzee-maven-plugin:3.12.1:repackage (package) on project eclipse-microprofile-kumuluz-test: Execution package of goal com.kumuluz.ee:kumuluzee-maven-plugin:3.12.1:repackage failed.: UnsupportedOperationException -> [Help 1]**

this started happening after I upgrade the build from java 16 to 17 at https://github.com/ozkanpakdil/test-microservice-frameworks/blob/master/.github/workflows/maven.yml#L17

here is my pom https://github.com/ozkanpakdil/test-microservice-frameworks/blob/master/eclipse-microprofile-kumuluz-test/pom.xml

can you tell me how to enforce mvn 3.6.3 ? and any idea when java 17 will be supported ?
Thanks
